gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -126,9 +130,9 @@ protected void removeToken(final String token) {
 
     try {
       aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME, token);
-      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME,token + "--max");
+      aliasService.removeAliasForCluster(AliasService.NO_CLUSTER_NAME,token + "TOKEN_MAX_LIFETIME_POSTFIX");
```

Comment:
This shouldn't be a string - it should be the variable reference?

Reply:
I don't follow, this is a variable reference, I took out the string and created a variable "TOKEN_MAX_LIFETIME_POSTFIX".

Reply:
Yea you have `TOKEN_MAX_LIFETIME_POSTFIX` as a string with `"TOKEN_MAX_LIFETIME_POSTFIX"`. Instead of just using `TOKEN_MAX_LIFETIME_POSTFIX`.

Reply:
<facepalm> and I was wondering my *--max tokens are not getting deleted. Thanks @risdenk !